plugins/atlas/skills/histfitter/references/config-api.md

```
@@ -0,0 +1,415 @@
+meas.addConstraintTerm("gamma_stat_SR_0", "Poisson")
+meas.addConstraintTerm("mu_BKG", "LogNormal", 0.1)  # with relUnc
+meas.addConstraintTerm("mu_BKG", "Uniform")
+meas.addConstraintTerm("mu_BKG", "NoConstraint")
+```
+
+Constraint types: `"Gaussian"`, `"Poisson"`, `"LogNormal"`, `"Uniform"`,
+`"NoConstraint"`.
+
+## Object hierarchy
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)